lib/mix/lib/mix/task.ex

```diff
@@ -7,11 +7,11 @@ defmodule Mix.Task do
   in a module starting with `Mix.Tasks.` and defining
   the `run/1` function:
 
-      defmodule Mix.Tasks.Hello do
+      defmodule Mix.Tasks.Echo do
         use Mix.Task
 
-        def run(_args) do
-          Mix.shell.info "hello"
+        def run(args) do
+          Mix.shell.info Enum.join(args, " ")
```

Comment:
@josevalim output could be confusing:
```
qwe qwe asd
```
was it run as `echo "qwe qwe" asd` or as `echo qwe qwe asd`?
It also doesn't help with understanding of what arguments will be in `echo 1 2`.
I think it quite unfriendly output for beginners.